### ISSUE TITLE
Installation instructions: please update the readme front page

### ISSUE THREAD

**Issue by m040601** [captn3m0/pystitcher#19]
Thanks for your work providing this tool. I ended up here looking for an alternative to python stapler.

There's a lot of good and important stuff on the README.

Please add one or two lines, at the top of the README with the most important stuff.
How to install it.

It might be obvious to you, a python developer, but not for a potential end user.
Is it using "pip install xyz" ? Will it work with pipx ? Are there any "official" packages for Linux distro xyz ?

Thanks in advance.

**Comment by m040601** [comment]
This is what I did on Arch Linux to quickly test it out:

```
python -m venv testing_pystitcher

cd ./testing_pystitcher

source bin/activate

pip install pypdf3

pip install pystitcher

```

**Comment by captn3m0** [comment]
Can you see if this looks good: https://github.com/captn3m0/pystitcher/wiki/Installation

I'll put a link in the README.

**Comment by m040601** [comment]
`pipx install pystitcher`

I had actually tried this as well.
But failed because I misspelled  "pystitcher" for "pysticher" and it failed.

Tested it again and it worked fine.
Pipx is a great option, if pystitcher is not packaged for your system, and you dont want to "pollute it" with a system wide "pip install".

> Can you see if this looks good

Yes it does.
Just add the "--user" advice. Much cleaner.

From:

> You can use pip to install pystitcher:
> pip install pystitcher

to someting like:


> You can use pip to install pystitcher:
> pip install pystitcher (system wide)
> pip install --user pystitcher (user)
